core/internal/gethwrappers2/go_generate.go

```diff
@@ -1,10 +1,4 @@
 // Package gethwrappers keeps track of the golang wrappers of the solidity contracts
 package main
 
-//go:generate ./compile.sh 15000 ../../../../libocr-internal/contract2/AccessControlledOCR2Aggregator.sol
-//go:generate ./compile.sh 15000 ../../../../libocr-internal/contract2/OCR2Aggregator.sol
-//go:generate ./compile.sh 15000 ../../../../libocr-internal/contract2/ExposedOCR2Aggregator.sol
-
-//go:generate ./compile.sh 1000 ../../../../libocr-internal/contract2/TestOCR2Aggregator.sol
-//go:generate ./compile.sh 1000 ../../../../libocr-internal/contract2/TestValidator.sol
-//go:generate ./compile.sh 1000 ../../../../libocr-internal/contract2/AccessControlTestHelper.sol
+//TODO how is OffchainAggregator generated?! https://smartcontract-it.atlassian.net/browse/BCF-1930
```

Comment:
Nothing being generated was used, and the only used package is not generated.

Reply:
IIRC OffchainAggregator is also obsolete, it's been replaced by a differently named contract in libocr.  So I think what we need is to import it from there, but adapt our tests to use the newer contract spec (what we use in production) instead of an ancient one.

Reply:
OK :ok_hand: To be clear, this PR is _not_ meant to address https://smartcontract-it.atlassian.net/browse/BCF-1930. It was previously blocked by that issue, but no longer is. Please note any relevant details over there :pray: